uber/config.py

```diff
@@ -106,18 +106,21 @@ def get_attendee_price(self, dt=None):
         price = self.INITIAL_ATTENDEE
         if self.PRICE_BUMPS_ENABLED:
 
+            for day, bumped_price in sorted(self.PRICE_BUMPS.items()):
+                if (dt or sa.localized_now()) >= day:
+                    price = bumped_price
+                    # If we set a price during the event, it should be used regardless of badge sales
+                    if c.EPOCH >= day >= c.ESCHATON:
+                        return price
+
             if dt or c.HARDCORE_OPTIMIZATIONS_ENABLED:
```

Comment:
I think we can now remove the ```if dt``` here to change this line to be just: ```if c.HARDCORE_OPTIMIZATIONS_ENABLED:```

i think? yea, pretty sure